test/test_other.py

```diff
@@ -112,6 +112,15 @@ def with_both_compilers(f):
   return f
 
 
+def with_debug_flags(f):
```

Comment:
With seems to run the test both with and without debug flags.

If that is desired then the name should really be `also_with_debug_flags`.

However, I think there is no need to run these tests twice.  Just add `-sDYLINK_DEBUG` to explictly to a test or two.  (no need for the =1 BTW)